tests/test_harvester_daily_toa_radative_flux.py

```diff
@@ -116,87 +116,115 @@ def test_global_mean_values(tolerance=0.001):
     assert data1[0].value >= (1 - tolerance) * global_mean
 
-'''temporarily commenting out the following 3 failing unit tests (https://github.com/NOAA-PSL/score-hv/issues/56)
-     
 def test_global_mean_values2(tolerance=0.001):
-    """This function tests the weighted means of each of the required variables.
-       The required variables[dswrf_avetoa, ulwrf_avetoa and uswrf_avetoa].
-       The weighted means are calculated in a separate program from the
-       files in the above bfg files and then hard coded in this function for a 
-       test of what is returned from the harvester.
     """
-    data1 = harvest(VALID_CONFIG_DICT)
-    
+       The weighted_means = [351.93690539738895, 242.84720151427342, 99.06752861939572]
+       were calculated for each of the required variables in a separate python script using the
+       bfg files listed above. 
+       This function calculates the weighted mean for each of the required variables
+       from the bfg files above.  The calculated weighted mean is then
+       compared to the hard coded value of the weighted mean for that variable.
+    """
     weighted_means = [351.93690539738895, 242.84720151427342, 99.06752861939572]
     weighted_means_array   = np.array(weighted_means)
+
+    gridcell_area_data = Dataset(GRIDCELL_AREA_DATA_PATH)
+    norm_weights = gridcell_area_data.variables['area'][:] / np.sum(
+                                        gridcell_area_data.variables['area'][:])
     num_vars = len(required_vars)
-    for i, harvested_tuple in enumerate(data1):
-        if harvested_tuple.statistic == 'mean':
-           for ivar in range(num_vars):
-               var_weight_mean = np.float32(weighted_means_array[ivar])
-               assert var_weight_mean <= (1 + tolerance) * harvested_tuple.value[ivar+1] 
-               assert var_weight_mean >= (1 - tolerance) * harvested_tuple.value[ivar+1]
-
-def test_gridcell_min_max(tolerance=0.001):
-    """
-      This function tests the minimum and maximum value 
-      of each of the required variables. 
-      required variables[dswrf_avetoa,ulwrf_avetoa and uswrf_avetoa].
-      The minimum and maximum values are calculated in a 
-      separate program from the bfg files above and then  
-      hard coded in this function for a 
-      test of what is returned from the harvester.
-    """
+    for ivar in range(num_vars): 
+        summation = 0.0
+        var_name = required_vars[ivar]
+        for file_count, data_file in enumerate(BFG_PATH):
+               test_rootgrp = Dataset(data_file)
+               with Dataset(data_file, 'r') as data:
+                    var_data = test_rootgrp.variables[var_name]
+                    summation += var_data[0]
+        value = summation/(file_count + 1) 
+        ivar_weighted_mean = np.ma.sum(norm_weights * value) 
+        assert ivar_weighted_mean <= (1 + tolerance) * weighted_means[ivar] 
+        assert ivar_weighted_mean >= (1 - tolerance) * weighted_means[ivar]
```

Comment:
is this the assert tolerance pattern that should be used in those duplicates above?